app/assets/javascripts/lifeitup_layout.js

```diff
@@ -61,7 +61,10 @@ function layout_resizer () {
 // init popovers
 
 $(function () {
-  $('[data-toggle="popover"]').popover()
+  $('body').popover({
+    selector: '[data-toggle="popover"]',
+    trigger: 'focus'

```

Comment:
I think @cyntss is right. We should try to get rid of all the lifeitup stuff we used for boostrapping portus.